content/chat/setup.textile

```diff
@@ -59,6 +60,11 @@ import * as Ably from 'ably';
 import ChatClient from '@ably/chat';
 ```
 
+```[react]
+import { ChatClientProvider } from '@ably/chat/react';
+import * as Ably from 'ably';
+```
+
 h3(#cdn). CDN
 
 Reference the Pub/Sub SDK and the Chat SDK within your HTML file:
```

Comment:
Worth hiding this section for React, as the React package is not available on the CDN?

Reply:
There's a bug that means it gets dropped from the in-page navigation if I do this, so I left it in originally. Will remove though.